### PR TITLE
Destination: Create application from VictorOps incident

### PR DESCRIPTION
**Pull request by cjwooo**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: destinations/airbyte-faros-destination/src/converters/victorops/incidents.ts [hunk 1]
```diff
@@ -162,30 +162,31 @@ export class Incidents extends VictorOpsConverter {
       appField in incident && typeof incident[appField] === 'string'
         ? incident[appField]
         : incident.service;
+    let application = {name: service, platform: ''};
 
     // if we have an app mapping specified, only then write the application and impact records
```
Comment (willmarks) [inline destinations/airbyte-faros-destination/src/converters/victorops/incidents.ts:167]: Comment no longer applies.

File: destinations/airbyte-faros-destination/src/converters/victorops/incidents.ts [hunk 2]
```diff
@@ -162,30 +162,31 @@ export class Incidents extends VictorOpsConverter {
...
+      record: application,
+    });
+
+    res.push({
+      model: 'ims_IncidentApplicationImpact',
+      record: {
+        incident: incidentRef,
+        application,
+      },
+    });
```
Comment (willmarks) [inline destinations/airbyte-faros-destination/src/converters/victorops/incidents.ts:189]: Interesting. I was under the impression that we only cared about applications in IMS when there was an explicit mapping. If we always care now, we will need to make a this change to all the other IMS connectors I think. Why do we want to do this?

**Review by willmarks — APPROVED** [review]

File: destinations/airbyte-faros-destination/src/converters/victorops/incidents.ts (hunk 2 above)
Comment (cjwooo) [reply destinations/airbyte-faros-destination/src/converters/victorops/incidents.ts:189]: Consistency for IMS-related sources https://faros-ai.atlassian.net/browse/FAI-1751. Pagerduty and Statuspage converters already create the application when there's no explicit mapping.

Comment (willmarks) [reply destinations/airbyte-faros-destination/src/converters/victorops/incidents.ts:188]: ah ok sounds good. I will make the same change for Datadog.

**Review by willmarks — APPROVED** [review]
Datadog looks good!

**Pull request merged**